internal/oci/runtime_oci.go

```diff
@@ -454,6 +454,21 @@ func (r *runtimeOCI) ExecSyncContainer(c *Container, command []string, timeout i
 
 	logrus.Debugf("Received container exit code: %v, message: %s", ec.ExitCode, ec.Message)
 
+	// This has to be in sync with the message returned by conmon
+	timedOutMessage := "command timed out"
```

Comment:
This should be defined in `https://github.com/containers/conmon/blob/master/runner/config/config.go` and then we can import it

Reply:
:+1: Opened https://github.com/containers/conmon/pull/110